### DISCUSSION TITLE
How to make quarto warn about missing figure file?

### DISCUSSION THREAD

**Discussion by langkabh** [quarto-dev/quarto-cli#10483] [Q&A] [answered]
### Description

Hi, I maintain a large website (hundreds of pages written by many different authors). I recently stumbled over the fact that `quarto render` won't warn me if any of the figure resources are missing. For example, a page might contain a reference

```
![](/img/tset.png)
```
while such a file doesn't exist in the /img folder and the correct file name would be `/img/test.png`.

Is there a way I can make quarto warn me about such unresolved references?

**Comment by mcanouil** [comment]
There is nothing to do.
If a resource is not found, it is printed at least when using `quarto preview` but indeed `quarto render` does not print the infos.
<img width="1624" alt="image" src="https://github.com/user-attachments/assets/5640f04c-d2c1-4949-ae52-73f70b542507">

**Reply by mcanouil** [reply, marked as answer]
I've open an issue to improve this:
- #10484

Meanwhile and depending on the format: `quarto render --embed-resources` could be use as a workaround to identify those missing resources.

**Reply by langkabh** [reply]
Many thanks for the quick response!

`quarto render --embed-resources` doesn't work for my actual rendering, but it does the job as a pre-release check and therefore addresses my particular need. Thanks for the hint!
